Package@swift-5.9.swift

```diff
@@ -0,0 +1,66 @@
+// swift-tools-version:5.9
```

Comment:
This file was added to not break support for older Swift versions.

See details about this here: https://www.polpiella.dev/version-specific-package-manifests